src/components/Main.js

```diff
@@ -0,0 +1,48 @@
+import Form from "./Form";
+import FactDisplay from "./FactDisplay";
+import {useState} from 'react';
+
+const Main = ()=>{   
+    const[fact,setFact]=useState("")
+    const[poster, setPoster]=useState("")
+    const[showName, setShowName]=useState("")
+    
+    let htmlRegexG = `<(?:"[^"]*"['"]*|'[^']*'['"]*|[^'">])+>`;
+    fact.replace(htmlRegexG, ''); 
+
```

Comment:
just commenting because this is so cool - was not sure how to implement Regex so nice to see it in action